main.c

```diff
@@ -1380,7 +1381,7 @@ int main(int argc, char **argv)
 		darray_append(handlers, tmp_handler);
 	}
 
-	tcmulib_context = tcmulib_initialize(handlers.item, handlers.size);
+	tcmulib_context = tcmulib_initialize(handlers.item, handlers.size, use_netlink);
```

Comment:
You can just discard the `use_netlink` and set it to `true` here.

Reply:
Done